README.rst

```diff
@@ -716,6 +716,19 @@ Choose from the list of available rules:
   - ``annotations`` (``array``): list of annotations to remove, e.g. ``["author"]``;
     defaults to ``[]``
 
+* **global_namespace_import**
+
+  Imports or fully qualifies global classes/functions/constants.
+
+  Configuration options:
+
+  - ``import_classes`` (``false``, ``null``, ``true``): whether to import, not import or
+    ignore global classes; defaults to ``null``
+  - ``import_constants`` (``false``, ``null``, ``true``): whether to import, not import or
+    ignore global constants; defaults to ``null``
+  - ``import_functions`` (``false``, ``null``, ``true``): whether to import, not import or
+    ignore global functions; defaults to ``null``
```

Comment:
At the moment the default value for all options is `null`. But this means the fixer wouldn't do anything without configuration.
This seems not to be allowed.
So which default value should we use here?

Reply:
There already are some fixers that can be configured in a way that does not really make sense. Usually there is no validation error in such case so maybe the current setup is fine.

Though maybe we could split the fixer into three new ones, which would allow to remove `null` from allowed configuration values.

Reply:
I get this error with phpunit:

> [global_namespace_import] Has no sample for default configuration.

But I can not add a sample for the default configuration, because samples without any change after fixing, are rejected by the tests.

So we have to change the defaults to `true` (or `false`), but we can still accept `null`.

Reply:
I would default to:
- `import_classes`: `true`
- `import_functions`: `null`
- `import_constants`: `null`

as this seems to be the most widespread way of using imports.